webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -603,6 +608,253 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
+        fontSize = 15;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_LEFT, "Attempt: __ of 3", competitorInfoLeft+padding, top-offsetTop, 0);
+        offsetTop += fontSize + marginBottom + (int) (marginBottom*1.8);
+        cb.endText();
+
+        cb.beginText();
+        fontSize = 15;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_LEFT, "Competitor: __________________", competitorInfoLeft+padding, top-offsetTop, 0);
+        offsetTop += fontSize + marginBottom + (int) (marginBottom*1.8);
+        cb.endText();
+
+        cb.beginText();
+
+        fontSize = 15;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_LEFT, "WCA ID:", competitorInfoLeft+padding, top-offsetTop, 0);
+
+        cb.setFontAndSize(bf, 19);
+        int wcaIdLength = 63;
+        cb.showTextAligned(PdfContentByte.ALIGN_LEFT, "_ _ _ _  _ _ _ _  _ _", competitorInfoLeft+padding+wcaIdLength, top-offsetTop, 0);
+
+        offsetTop += fontSize + (int) (marginBottom*2.4);
+        cb.endText();
+
+
+        cb.beginText();
+        fontSize = 11;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "DO NOT FILL IF YOU ARE THE COMPETITOR", competitorInfoLeft + (right-competitorInfoLeft)/2, top-offsetTop, 0);
+        offsetTop += fontSize + marginBottom;
+        cb.endText();
+
+        cb.beginText();
+        fontSize = 11;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "Graded by: _______________ Result: ______", competitorInfoLeft + (right-competitorInfoLeft)/2, top-offsetTop, 0);
+        offsetTop += fontSize + (marginBottom*5);
+        cb.endText();
+
+        cb.beginText();
+        fontSize = 11;
+        cb.setFontAndSize(bf, fontSize);
+        cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "Please see seperate sheet for scrambles.", competitorInfoLeft + (right-competitorInfoLeft)/2, top-offsetTop, 0);
+        offsetTop += fontSize + marginBottom;
+        cb.endText();
+
+        cb.beginText();
+        cb.setFontAndSize(bf, 25f);
+        int MAGIC_NUMBER = 40; // kill me now
+        cb.showTextAligned(PdfContentByte.ALIGN_CENTER, "Fewest Moves", left+(competitorInfoLeft-left)/2, top-MAGIC_NUMBER, 0);
+        cb.endText();
+
+        com.itextpdf.text.List rules = new com.itextpdf.text.List(com.itextpdf.text.List.UNORDERED);
+        rules.add("Notate your solution by writing one move per bar.");
+        rules.add("To delete moves, clearly erase/blacken them.");
+        rules.add("Face moves F, B, R, L, U, and D are clockwise.");
+        rules.add("Rotations x, y, and z follow R, U, and F.");
+        rules.add("' inverts a move; 2 doubles a move. (e.g.: U', U2)");
+        rules.add("w makes a face move into two layers. (e.g.: Uw)");
+        rules.add("A [lowercase] move is a cube rotation. (e.g.: [u])");
```

Comment:
I think it's worth trying to share the code because when/if these rules change, I don't want to have to remember to edit them in two places.